gradle/instrumentation.gradle

```diff
@@ -55,3 +51,76 @@ afterEvaluate {
     project(':javaagent-tooling').configurations.instrumentationMuzzle + configurations.runtimeClasspath
   ).configure()
 }
+
+configurations {
+  testInstrumentation
+}
+
+shadowJar {
+  configurations = [project.configurations.runtimeClasspath, project.configurations.testInstrumentation]
+  mergeServiceFiles()
+
+  archiveFileName = 'agent-testing.jar'
+
+  dependencies {
+    exclude(project(':javaagent-bootstrap'))
+    exclude(project(':instrumentation-api'))
+    exclude(project(':javaagent-api'))
+    exclude(project(':javaagent-tooling'))
```

Comment:
These are `compileOnly`, do we still have to exclude them?

Reply:
removed